isis/src/mex/apps/hrsc2isis/hrsc2isis.cpp

```diff
@@ -15,12 +15,11 @@ find files of those names at the top level of this repository. **/
 #include "LineManager.h"
 #include "IException.h"
 #include "Application.h"
+#include "OriginalLabel.h"
 
 using namespace std;
 namespace Isis{
-  void ImportHrscStereoImage(ProcessImportPds &p, Pvl &label);
-  void ImportHrscStereoImage(ProcessImportPds &p, Pvl &label, UserInterface &ui);
-  void ImportHrscSrcImage(ProcessImportPds &p, Pvl &label);
```

Comment:
The non UI versions were not really used, or rather, the UI version was used for stereo but not for non-stereo which meant output atts were not applied depending on detector ID as well. Figure I'd normalize them and only use the versions with UI as the input and apply out attrs to both.